src/panels/config/integrations/integration-panels/zwave_js/zwave_js-node-config.ts

```diff
@@ -421,6 +428,12 @@ class ZWaveJSNodeConfig extends SubscribeMixin(LitElement) {
       return;
     }
 
+    this._nodeMetadata = await fetchNodeMetadata(
```

Comment:
Use `await Promise.all()` here to do both the meta and config calls at the same time